R/create.R

```diff
@@ -240,22 +240,27 @@ check_not_nested <- function(path, name) {
   invisible()
 }
 
-rationalize_fork <- function(fork, repo_info, pat_available, user = NULL) {
-
+rationalize_fork <- function(fork, repo_info, auth_token) {
```

Comment:
I think this function would be a little clearer if you stored `have_github_token(auth_token)` in a variable